conf/messages.cy

```diff
@@ -60,7 +60,7 @@ accessibility.compliance.noncompliant=Nid yw’r gwasanaeth hwn yn cydymffurfio
 accessibility.compliance.noncompliant.app=Nid yw’r ap hwn yn cydymffurfio â
 accessibility.compliance.noncompliant.explanation=. Nid yw’r gwasanaeth hwn wedi cael ei wirio eto i weld a yw’n cydymffurfio neu beidio, felly gall rhai defnyddwyr ei chael yn anodd defnyddio rhannau o’r gwasanaeth.
 accessibility.compliance.noncompliant.explanation.app=. Nid yw’r ap hwn wedi cael ei wirio eto i weld a yw’n cydymffurfio neu beidio, felly gall rhai defnyddwyr ei chael yn anodd defnyddio rhannau o’r ap.
-accessibility.guidelines=safon ‘AA’ Canllawiau Hygyrchedd Cynnwys y We, fersiwn {0}
+accessibility.guidelines=Chanllawiau Hygyrchedd Cynnwys y We fersiwn {0} safon AA
```

Comment:
is this change not applicable for english?

Reply:
its already as per the Design Acceptance Criteria